### DISCUSSION TITLE
Root element is missing

### DISCUSSION THREAD

**Discussion by PompeyLiang** [visose/Robots#131] [Q&A] [answered]
I have some problem in installing the plug-in.

I download the plug-in file and the robot model with the corresponding .xml file, unlocked and put them in the right path also.  

But in grasshopper all the components are there and work well, just the load robot system doesn't work and shows "Root element is missing". 
The load robot system was select and place in the canvas without the value list that contain the robot library. is that the problem?

**Comment by visose** [comment]
A common mistake is to download the html page instead of the actual xml file. If you wait to tomorrow there will be a new version that allows you to install the library inside Grasshopper, so you don't have to download and copy files manually.

**Comment by visose** [comment]
@PompeyLiang I just published a new version, after installing click on the "Libraries" button of the Load robot system component. Delete any files you have copied previously to your Documents\Robots folder.

**Reply by PompeyLiang** [reply]
Thanks visose, I well try now and let you know whether it work or not

**Reply by PompeyLiang** [reply]
I still have this problem.
Do you think is the problem of my latop's system setting? So the component can't find the correct path that store the robot library file?

Here is the screenshot
![Screenshot (30)](https://user-images.githubusercontent.com/38983015/151059251-5001b729-88d1-4db5-8494-efe9e40f782f.png)

**Reply by visose** [reply]
You have to first manually delete the files you copied originally, then try installing it again using this UI.

**Comment by visose** [comment, marked as answer]
Closing this as resolved. `Root element is missing` usually means one of the local library XML files is invalid or incomplete, often because an HTML page or partial download was saved as `.xml`. Delete the manually copied/broken files from the local Robots library folder and install/download libraries through the Libraries UI, or make sure each `.xml` is valid XML and has its matching `.3dm` file.
